README.md

```diff
@@ -1,2 +1,27 @@
-# optee_examples
-OP-TEE Sample Applications
+# OP-TEE Sample Applications
```

Comment:
I agree, but I think that in the root `README.md` we should have at least some contents with short description for each example